flink-connectors/flink-sql-connector-hive-3.1.2/pom.xml

```diff
@@ -69,10 +69,56 @@ under the License.
 			<artifactId>antlr-runtime</artifactId>
 			<version>3.5.2</version>
 		</dependency>
+
+		<!-- hadoop deps to make the copied code compile -->
+		<dependency>
+			<groupId>org.apache.hadoop</groupId>
+			<artifactId>hadoop-mapreduce-client-core</artifactId>
+			<version>2.8.2</version>
```

Comment:
use Flink's Hadoop version variable?

Reply:
Maybe it's better to compile against the same hadoop version that hive-3.1.2 depends upon (which is 3.1.0). Otherwise there can be incompatibility issues at runtime. WDYT? @twalthr

Reply:
I leave this decision to the Hive experts. Fine with me.